cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -5,10 +5,12 @@
 from cfme.fixtures.provider import (dual_network_template, dual_disk_template,
  dportgroup_template, win7_template, win10_template, win2016_template, rhel69_template,
  win2012_template, ubuntu16_template, rhel7_minimal)
+from cfme.fixtures.v2v import _form_data
```

Comment:
I am reusing KK's function here, there is no reason behind `_` as per I know. We can also use `as` in case, what say @digitronik  
``from cfme.fixtures.v2v import _form_data as form_data``